src/api/app/views/webui2/webui/attribute/index.html.haml

```diff
@@ -0,0 +1,48 @@
+- @pagetitle = "Attributes of #{@package} (Project #{@project})"
+- package_bread_crumb 'Attributes'
+
+.card
+  = render(partial: 'webui/package/tabs', locals: { project: @project, package: @package })
+
+  .card-body
+    %h3= @pagetitle
+    - if @attributes.present?
+      .table-responsive
+        %table.table.table-hover#attributes
+          %thead.thead-light
+            %tr
+              %th.col-auto{ scope: 'col' } Attributes
+              %th.col{ scope: 'col' } Values
+              - if policy(@package).update?
+                %th.col-auto{ scope: 'col' } Actions
+          %tbody
+            - @attributes.each do |attribute|
+              %tr
+                %td= attribute.fullname
+                %td
+                  %ul
+                    - unless attribute.values.empty?
+                      - attribute.values.each do |v|
+                        %li.value
+                          %pre.d-inline= v.value
+                    - unless attribute.issues.empty?
+                      %li= attribute.issues.map(&:name).join(', ')
```

Comment:
[nitpick] we could also use [to_sentence](https://apidock.com/rails/Array/to_sentence) here instead of the join :thinking: